sklearn/covariance/_graph_lasso.py

```diff
@@ -561,6 +561,9 @@ class GraphicalLassoCV(GraphicalLasso):
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
+        ..versionchanged:: v0.20
```

Comment:
Maybe discussed somewhere but I haven't found it. Is it normal that you don't have a space after `..` whereas you have one for versionadded ?

Reply:
I think it's a relic of my attempts to fix the Unexpected indentation error. I think I should better add spaces after `..` everywhere.